src/components/inputs/money-input/money-input.js

```diff
@@ -218,6 +218,8 @@ export default class MoneyInput extends React.Component {
     return moneyValue && moneyValue.type === 'highPrecision';
   };
 
+  static isTouched = touched => touched && Object.values(touched).some(Boolean);
```

Comment:
Moved it up so that all static methods used by consumers are grouped at the top.